.github/workflows/build_and_test.yml

```diff
@@ -24,22 +24,30 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10"]
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Pip installation
       run: |
         python -m pip install --upgrade pip
+        pip install .
```

Comment:
What has changed to before, if installing wheel and dictdiffer prior to pip install . used to be necessary?

Reply:
wheel is not necessary at this stage, the `dictdiffer` dependency was unused